docs/examples/DataSet/Offline Plotting Tutorial.ipynb

```diff
@@ -11019,26 +11026,16 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "By default Matplotlib renders each individual data point as a separate square in 2D plots when storing in a vector format (pdf,svg). This is not a problem for small data sets, but the time needed to generate a pdf increases rapidly with the number of data points. Therefore, `plot_dataset` will automatically rasterize the data (lines, ticks and labels are still stored as text) if more than 5000 data points are plotted. The particular value of the rasterization threshold can be set in the `qcodesrc.json` config file.\n",
+    "By default `matplotlib` renders each individual data point as a separate square in 2D plots when storing in a vector format (pdf,svg). This is not a problem for small data sets, but the time needed to generate a pdf increases rapidly with the number of data points. Therefore, the `plot_dataset` will automatically rasterize the data (lines, ticks and labels are still stored as text) if more than 5000 data points are plotted. The particular value of the rasterization threshold can be set in the `qcodesrc.json` config file.\n",
     "\n",
-    "Alternatively the rasterized keyword can be passed to the `plot_dataset` function`"
+    "Alternatively the rasterized keyword can be passed to the `plot_dataset` function."
    ]
   },
   {
-   "cell_type": "code",
-   "execution_count": 64,
+   "cell_type": "markdown",
    "metadata": {},
-   "outputs": [
-    {
-     "name": "stdout",
-     "output_type": "stream",
-     "text": [
-      "Starting experimental run with id: 30\n"
-     ]
-    }
-   ],
    "source": [
-    "meas = Measurement(exp=exp)\n",
+    "## meas = Measurement(exp=exp)\n",
```

Comment:
why commenting out?

Reply:
I believe this is a mistake that I did unintentionally. I will check it and roll it back.